packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -941,6 +950,7 @@ class IndexedStack extends MultiChildRenderObjectWidget {
   }
 }
 
+/// Metadata for children of [Stack].
```

Comment:
This needs more detail, like the fact that it has to not have any other rendering widgets between it and the stack.